schemas/com.youtube/youtube/jsonschema/1-0-0

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a youtube player event",
+  "self": {
+    "vendor": "com.youtube",
+    "name": "youtube",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "avaliablePlaybackRates": {
+      "type": "array",
+      "description": "An array of playback rates in which the current video is available",
+      "items": {
+        "type": "number",
+        "minimum": 0,
+        "maximum": 128
+      }
+    },
+    "avaliableQualityLevels": {
+      "type": ["array", "null"],
+      "description": "An array of quality levels in which the current video is available",
+      "items": {
+        "type": "string",
+        "maxLength": 128
+      }
+    },
+    "cued": {
+      "type": "boolean",
+      "description": "If the video is cued"
+    },
+    "playerId": {
+      "type": "string",
+      "description": "The HTML id of the video element",
+      "maxLength": 65535
+    },
+    "autoPlay": {
+      "type": "boolean",
+      "description": "This specifies whether the initial video will automatically start to play when the player loads."
+    },
+    "buffering": {
+      "type": "boolean",
+      "description": "If the player is buffering"
+    },
+    "controls": {
+      "type": "boolean",
+      "description": "Whether the video player controls are displayed"
+    },
+    "error": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A string of the latest error to occur, or null if no errors",
+      "enum": [
+        "INVALID_PARAMETER",
+        "HTML5_PLAYER_ERROR",
+        "NOT_FOUND",
+        "EMBED_DISALLOWED"
+      ]
+    },
+    "loaded": {
+      "type": "number",
```

Comment:
Seems other percentage properties in other schemas are `integer`. Presumably this one is an exception?

Reply:
Good catch, changed to `integer`.